### PR TITLE
[Term Entry] PHP Arrays array_column()

### PR DESCRIPTION
**Pull request by wierci**
### Description

- Add the term entry for the array_splice method in PHP
- Closes issue #2876 

### Type of Change

- Adding a new entry

### Checklist

- [x ] All writings are my own.
- [x ] My entry follows the Codecademy Docs style guide.
- [x ] My changes generate no new warnings.
- [x ] I have performed a self-review of my own writing and code.
- [x ] I have checked my entry and corrected any misspellings.
- [x ] I have made corresponding changes to the documentation if needed.
- [x ] I have confirmed my changes are not being pushed from my forked `main` branch.
- [x ] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [x ] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

### CODE REVIEW

File: content/php/concepts/arrays/terms/array-column/array-column.md [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+array_column(array, column_key, index_key)
+```
+
+- `array`: A multi-dimensional array to use.
+- `column_key`: An integer or string key of the column of values to return. It may also be `null` to return complete arrays or objects.
+- `index_key`: The column to use as the index for the returned array.
+
+## Example
+
+In this example, the first names from an array are returned.
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-column/array-column.md:30]:
```suggestion
In this example, the `first_name` keys are returned using the `array_column()` method.
```

File: content/php/concepts/arrays/terms/array-column/array-column.md [hunk 2]
```diff
@@ -0,0 +1,120 @@
...
+```shell
+Array
+(
+  [0] => Jane
+  [1] => Lois
+  [2] => Lola
+)
+```
+
+Additionally, the returened values can be indexed by the "id" column:
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-column/array-column.md:68]:
```suggestion
Additionally, the returned values can be indexed by the `id` column:
```

File: content/php/concepts/arrays/terms/array-column/array-column.md [hunk 3]
```diff
@@ -0,0 +1,120 @@
...
+    'id' => 921,
+    'first_name' => 'Lois',
+    'last_name' => 'Griffin',
+  ),
+  array(
+    'id' => 475,
+    'first_name' => 'Lola',
+    'last_name' => 'Bunny',
+  )
+);
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-column/array-column.md:88]: I think we don't need to duplicate this part and we can just show the method with the new arguments

File: content/php/concepts/arrays/terms/array-column/array-column.md [hunk 4]
```diff
@@ -0,0 +1,120 @@
...
+```
+
+The output looks like this:
+
+```shell
+Array
+(
+  [0] => Jane
+  [1] => Lola
+  [2] => Lois
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-column/array-column.md:102]: this looks like it's missing the ids

File: content/php/concepts/arrays/terms/array-column/array-column.md [hunk 5]
```diff
@@ -0,0 +1,120 @@
...
+)
+```
+
+## Codebyte Example
+
+The following codebyte example declares a `$colorFabric`. It then returns all fabric types in a new array called `$fabrics`.
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-column/array-column.md:108]:
```suggestion
The following codebyte example is runnable and utilizes the `array_column()` method.
```

File: content/php/concepts/arrays/terms/array-column/array-column.md [hunk 6]
```diff
@@ -0,0 +1,120 @@
...
+
+```codebyte/php
+<?php
+  $colorFabrics = array(
+  array('color' => 'white', 'fabric' => 'cotton'),
+  array('color' => 'blue', 'fabric' => 'denim'),
+  array('color' => 'beige', 'fabric' => 'linen'))
```
Comment (caupolicandiaz) [inline content/php/concepts/arrays/terms/array-column/array-column.md:115]:
```suggestion
  array('color' => 'beige', 'fabric' => 'linen')
```

**Review by caupolicandiaz — CHANGES_REQUESTED** [review]
@wierci I've completed my initial review. Please update the entry, and tag me when you've finished making changes. Thanks

**Comment by wierci** [comment]
@caupolicandiaz I've updated the entry and finished making changes.

**Review by SSwiniarski — APPROVED** [review]
@wierci Looks good.

**Pull request merged**